### ISSUE TITLE
Not showing only nature pictures anymore?

### ISSUE THREAD

**Issue by jancborchardt** [jancborchardt/unsplash#40]
Recently I got a bunch of pretty much non-nature pictures in the mix:

![unsplash not nature](https://user-images.githubusercontent.com/925062/51708067-36645500-2023-11e9-978f-71fdf382eccc.png)

![unsplash not nature face](https://user-images.githubusercontent.com/925062/51708065-36645500-2023-11e9-8482-e1b8d75a154b.png)

![unsplash not nature city](https://user-images.githubusercontent.com/925062/51708066-36645500-2023-11e9-912d-bc35b9946c60.png)

Can you reproduce this @newhinton @marius-wieschollek @Madh93? Can’t see any change on the API documentation at https://source.unsplash.com/

**Comment by marius-wieschollek** [comment]
Yes, i can confirm this.

**Comment by CHazz** [comment]
i have this problem now : 
https://images.unsplash.com/source-404?fit=crop&fm=jpg&h=800&q=60&w=1200

**Comment by newhinton** [comment]
Yup, i can confirm that. It seems they either changed the api and didn't update the documentation, or the api is temporarily broken. I'd suggest to monitor this, and if it doesn't start working again, we need to find out what exactly causes this if there is no statement from unsplash


it seems
https://source.unsplash.com/1600x900/?nature,water
isnt working, but
https://source.unsplash.com/1600x900/ is

**Comment by newhinton** [comment]
I will take a look at the wikimedia commons api in the next few days, maybe it's time to switch.

**Comment by tchibomann** [comment]
Yeah, noticed that too today. I've changed the header.css temporarily to the URL without get params. Works for now :D

**Comment by nate2014jatc** [comment]
I'd be glad to make an edit if it fixes this. As pretty as their 404 page is, it gets boring after a while XD

**Comment by nate2014jatc** [comment]
It's really strange that their API is failing on their end as well

**Comment by geramaya** [comment]
> 
> 
> I'd be glad to make an edit if it fixes this. As pretty as their 404 page is, it gets boring after a while XD
> 
> It's really strange that their API is failing on their end as well, has anyone submitted a bug report to them?

A bug at Unsplash or the nc-plugin and the api call?

**Comment by nate2014jatc** [comment]
Seems to be at Unsplash themselves. 
@newhinton discovered this, and I tried it out myself as well.
> it seems
>https://source.unsplash.com/1600x900/?nature,water
>isnt working, but
>https://source.unsplash.com/1600x900/ is

**Comment by geramaya** [comment]
> 
> 
> @nate2014jatc
> 
> You need to edit the following files:
> 
> unsplash/css/login.css
> unsplash/css/header.css
> 
> There is a line:
> 
> `background-image : url('https://source.unsplash.com/random/featured/?nature') !important;`
> 
> you can change the source of the url there:
> 
> ` background-image : url('https://source.unsplash.com/1920x1080/ ') !important;`
> 
> this shoud give you some random images

works! thx

**Comment by felixkick** [comment]
I'm having the same problem - 404-style-background. But as it is a hosted nextcloud-instance, I can't access the css-files.

Would it be possible to create an option in the NextCloud Admin Settings tabs under "Theming": 

API-Call for random image ---> Textbox

This way, you could also be more specific in the collections you want to see.

**Comment by geramaya** [comment]
> 
> 
> I'm having the same problem - 404-style-background. But as it is a hosted nextcloud-instance, I can't access the css-files.
> 
> Would it be possible to create an option in the NextCloud Admin Settings tabs under "Theming":
> 
> API-Call for random image ---> Textbox
> 
> This way, you could also be more specific in the collections you want to see.

Hm, I think you have to wait for a patch version :(

**Comment by nate2014jatc** [comment]
@felixkick @geramaya I sent an email to unsplash reporting the issue. I'll report back when(if) they get back to me.

**Comment by felixkick** [comment]
Found a little workaround myself: The App "Custom CSS" allows to bring in some own CSS-lines. I added

    body#body-login {
     	background-image : url('https://source.unsplash.com/collection/1751826') !important;
    }

and it worked again. Thanks !!!

**Comment by nate2014jatc** [comment]
I'm not holding my breath for a response. I'm really glad it got fixed though!

**Comment by nate2014jatc** [comment]
Glad I didn't hold my breath!

Here's the reply;
```
Hi Nate. Yes, we did have some issues with the API's random-search functionality the other day, however a fix has already be deployed. If you are still seeing this issue please let me know.

Thanks. 
Aaron
--
Find answers quickly in our Help Center
Chat to us at chat.unsplash.com
```

**Comment by AlexG-78** [comment]
it starts to happen again since yesterday

**Comment by e-mojo** [comment]
Same here, today.

**Comment by melMass** [comment]
It's back

**Comment by ktivdv** [comment]
yes it is  =(

> It's back

**Comment by dragonpil** [comment]
its back

**Comment by newhinton** [comment]
Please see #115
